linear-base.cabal

```diff
@@ -23,6 +23,7 @@ library
     Control.Optics.Linear.Internal
     Data.Bifunctor.Linear
     Data.Functor.Linear
+    Data.Semigroup.Linear
```

Comment:
Actually: rename this `Monoid`.

Reply:
Is there a particular reason this is better? I picked `Semigroup` since it matches the pattern in base where importing Semigroup also brings into scope Monoids too

Reply:
Just that `Monoid` is a more common structure than `Semigroup`. So it's a better namespace.